rackcity/models/change_plan.py

```diff
@@ -0,0 +1,22 @@
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class ChangePlan(models.Model):
+    name = models.CharField(max_length=150)
+    execution_time = models.DateTimeField(blank=True, null=True)
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ['name']
+        constraints = [
+            models.UniqueConstraint(
+                fields=['name', 'owner'],
```

Comment:
This says that two different users can make change plans with the same name - is that what we want?

Reply:
yeah - same idea as above comment, only individuals can see their own change plans